src/main/kotlin/exercise3/task1/BalancedBrackets.kt

```diff
@@ -26,9 +28,41 @@ package exercise3.task1
 
 
 internal fun isExpressionBalanced(expression: String): Boolean {
-    TODO("Implement me!!!")
+
+    if (expression.length %2 == 1) return false
+
+    val pair1 = '(' to ')'
+    val pair2 = '[' to ']'
+    val pair3 = '{' to '}'
+    val pairs = arrayOf(pair1, pair2, pair3)
```

Comment:
nit: It's cleaner to define array as following:
```arrayOf('(' to ')', '[' to ']', '{' to '}')```
since pair1, pair2 and pair3 aren't used later in the code.